content/blog/1.0-announcement/index.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Open GitOps 1.0 is finally here and why you should care
+date: "2021-11-01T09:00:01.284Z"
+description: "Open GitOps 1.0 brings a mature standard to the DevOps industry."
+author: Dan Garfield and ?
+thumbnail: images/GitOps-1.0.jpg
+---
+
+GitOps is a term coined by Alexis Richardson a few years ago and has grown immensely in popularity. In a nutshell, GitOps offers a new pattern for deploying software that differs from the traditional CD pipeline approach that has dominated the industry for decades. Last year, Amazon, Azure, Codefresh, Github, Redhat, and Weaveworks teamed up to launch the GitOps Working Group under the CNCF. The goal was to formalize all the ideas that have developed around GitOps into a single cohesive set of principles that would be easily digestible and repeatable by any organization.
+
+  
+
+We’re very proud to announce that over 60 companies, 96 interested parties, and 34 co-authors have brought the best of DevOps into the [Open GitOps 1.0 principles](https://github.com/open-gitops/documents/releases/tag/v1.0.0). Each principle discusses the desired state of a system and how it should be operated.
+
+  
+
+The desired state of a GitOps managed system must be:
+
+  
+
+1. **Declarative**
+A [system](https://github.com/open-gitops/documents/blob/main/GLOSSARY.md#software-system) managed by GitOps must have its desired state expressed [declaratively](https://github.com/open-gitops/documents/blob/main/GLOSSARY.md#declarative-description).
+
+2. **Versioned and Immutable**
+Desired state is [stored](https://github.com/open-gitops/documents/blob/main/GLOSSARY.md#state-store) in a way that enforces immutability, versioning and retains a complete version history.
+
+3. **Pulled Automatically**
+Software agents automatically pull the desired state declarations from the source.
+
+4. **Continuously Reconciled**
+Software agents [continuously](https://github.com/open-gitops/documents/blob/main/GLOSSARY.md#continuous) observe actual system state and [attempt to apply](https://github.com/open-gitops/documents/blob/main/GLOSSARY.md#reconciliation) the desired state.
+
+  
+
+## GitOps is more than just storing infrastructure-as-code
+
+As the [git history proves](https://github.com/open-gitops/documents/commits/main), the wording of each of these principles was very carefully chosen. While many of these principles should be familiar to DevOps professionals, many are surprised to learn that they still have work to do to implement GitOps fully.
```

Comment:
@todaywasawesome I thought I had commented on this earlier, but maybe it got lost.

The terminology "DevOps professionals" feels awkward in this context about GitOps principles.

Some professionals who are led by DevOps principles of shared ownership, workflow automation, and rapid feedback, may even say that DevOps is a philosophy, and DevOps isn't a job title at all.

Since your target audience includes those professionals, I suggest not using the terminology "DevOps professionals" here.

Usage is similar to how "CI/CD" would be clearer when written as "CI & CD", to clarify that CI is a development-focused practice, and CD is an operations-focused practice.  CI & CD are separate practices.

If a blog says "DevOps professional" and "CI/CD", then the professionals who you are trying to reach may tune out the blog.

Semantically, a blog is clearer to the intended audience by writing "professional led by DevOps principles", and "CI & CD".

In stark contrast, if a blog's target audience are recruiters using an Applicant Tracking System (ATS) to search keywords, then an ATS system might index "DevOps professional" and "CI/CD" as score-able keywords.

```suggestion
As the [git history proves](https://github.com/open-gitops/documents/commits/main), the wording of each of these principles was very carefully chosen. While many of these principles should be familiar to professionals, many are surprised to learn that they still have work to do to implement GitOps fully.
```

Reply:
Fair.